README.md

```diff
@@ -118,9 +118,16 @@ There is support for remote connections via commando plugin but you cannot use t
 We provide a standalone statically compiled binary. The targets are `linux` (amd64 build for GNU/Linux), `rasp` (aarch64 build for GNU/Linux) suitable for installation on your Raspberry PI and `darwin` (amd64 build for MacOS).
 If you need some different architecture that is no problem except that you will have to build the binaries yourself (so you will need to install Golang). You can also open an issue / pull request to extend the defaults (or provide better naming).
 
+### Nix
+
```

Comment:
I would prefer to start with `script` part and then nix option. I believe it is not as popular